pkg/types/alpine/v0.0.1/entry.go

```diff
@@ -322,14 +322,21 @@ func (v V001Entry) CreateFromArtifactProperties(ctx context.Context, props types
 
 	re.AlpineModel.PublicKey = &models.AlpineV001SchemaPublicKey{}
 	publicKeyBytes := props.PublicKeyBytes
-	if publicKeyBytes == nil {
-		publicKeyBytes, err = ioutil.ReadFile(filepath.Clean(props.PublicKeyPath.Path))
-		if err != nil {
-			return nil, fmt.Errorf("error reading public key file: %w", err)
+	if len(publicKeyBytes) == 0 {
+		if len(props.PublicKeyPath) > 1 {
```

Comment:
nit: for simplicity just use `if len(props.PublicKeyPath) != 1`? then no need for the second if clause

Reply:
done.